asadbek/db.sql

```diff
@@ -0,0 +1,188 @@
+1. Qo'shimcha jadvallar yaratish:
+
+university=# create table courses(
+university(# course_id serial primary key,
+university(# course_name varchar(100),
+university(# credit_hours int);
+CREATE TABLE
+
+university=# create table enrollements(
+university(# enrollment_id serial primary key,
+university(# student_id int,
+university(# foreign key (student_id) references students(student_id),
+university(# course_id int,
+university(# foreign key (course_id) references courses(course_id),
+university(# grade int);
+CREATE TABLE
```

Comment:
please don't write like this:
university=# create table enrollements(
university(# enrollment_id serial primary key,
university(# student_id int,
university(# foreign key (student_id) references students(student_id),
university(# course_id int,
university(# foreign key (course_id) references courses(course_id),
university(# grade int);
CREATE TABLE

instead you can write like this:
CREATE TABLE if not exists enrollments (
    enrollment_id SERIAL PRIMARY KEY,
    student_id INTEGER REFERENCES students(student_id),
    course_id INTEGER REFERENCES courses(course_id),
    grade INTEGER
);